tests/headingscommand.js

```diff
@@ -53,6 +53,15 @@ describe( 'HeadingsCommand', () => {
 				expect( command.value ).to.equal( format );
 			} );
 		}
+
+		it( 'should be equal to defaultFormat if format has not been found', () => {
```

Comment:
This is because when you focus selection inside a model element that is not in `formats` array, you get this error:
`Uncaught TypeError: Cannot read property 'label' of undefined`.

This is because no option has been found to be displayed in the dropdown.

However, as I think of it now, it should probably be done like it is in CKE4. There is "Format" option there whenever you are inside something that is not recognized by `Headings` feature. Right now, in CKE5, you will have "Paragraph" option displayed. Because of that you are unable to actually change to the paragraph using `Headings` dropdown.

I think this is worthy of seperate issue because then we have to ask ourselves what is the `defaultValue` -- still `paragraph` or maybe "format"?